src/dynd/types/categorical_type.cpp

```diff
@@ -469,19 +469,6 @@ struct get_ints_kernel : nd::base_kernel<get_ints_kernel> {
   }
 };
 
-static const std::map<std::string, nd::callable> &categorical_array_properties()
```

Comment:
Categorical types are broken, right now. They were too integrated with a variety of things we wanted to remove, so I disabled them. Meaning don't worry for now.